torch/nested/_internal/nested_tensor.py

```diff
@@ -134,53 +134,29 @@ def __reduce_ex__(self, proto):
     def __tensor_flatten__(self):
         ctx = {
             "requires_grad": self.requires_grad,
-            "ragged_size": self._size[self._ragged_idx],
```

Comment:
no longer need this in the context, thanks to `outer_size` being passed in to the unflatten func